srcpkgs/pounce/template

```diff
@@ -1,14 +1,15 @@
 # Template file for 'pounce'
 pkgname=pounce
-version=2.3
-revision=2
+version=2.4
+revision=1
 build_style=gnu-configure
 make_build_target="all"
 hostmakedepends="pkg-config"
-makedepends="libtls-devel"
+makedepends="libtls-devel openssl-devel"
```

Comment:
Why have `openssl-devel`?

Reply:
Falling back to /bin/openssl is fine on void, removed.